doc/zmq.md

```diff
@@ -46,11 +46,10 @@ operation.
 
 ## Enabling
 
-By default, the ZeroMQ feature is automatically compiled in if the
-necessary prerequisites are found.  To disable, use -DWITH_ZMQ=OFF
-during the *configure* step of building bitcoind:
+By default, the ZeroMQ feature is not automatically compiled.
+To enable, use `-DWITH_ZMQ=ON` when building:
 
-    $ cmake -B build -DWITH_ZMQ=OFF (other options)
+    $ cmake -B build -DWITH_ZMQ=ON (other options)
```

Comment:
I also noticed the same.
this line should be executable code, could we either remove `(other options)` or add it as a comment instead?

Reply:
Thanks for taking a look.  Decided to remove "(other options)" since it should be pretty clear to readers that the presence of this option wouldn't preclude the use of other ones needed.